pkg/cortex/cortex.go

```diff
@@ -219,9 +222,8 @@ type Cortex struct {
 	StoreGateway *storegateway.StoreGateway
 	MemberlistKV *memberlist.KVInitService
 
-	// Queryables that the querier should use to query the long
-	// term storage. It depends on the storage engine used.
-	StoreQueryables []querier.QueryableWithFilter
+	Queryable prom_storage.SampleAndChunkQueryable
```

Comment:
Why this change? Could you share more details about why this change is needed, please?